style.css

```diff
@@ -46,8 +50,29 @@ body {
 .contact-me .section-content input {
   width: 100%;
 }
-.contact-me .section-content input[name=message] {
+.contact-me .section-content textarea[name=message] {
   height: 200px;
+  width: 100%;
+}
+.contact-me .section-content .submit-button,
+.download a,
+.github button,
```

Comment:
I think this one should be a link instead of a button, because it will open in other place